### ISSUE TITLE
Update value of Viessmann parameters via MQTT?

### ISSUE THREAD

**Issue by Pampersfr** [sarnau/InsideViessmannVitosoft#12]
Hello

do you implemented the possibility to update parameters of viessmann via your viessmann2MQTT.py ?
for example : publish "setBetriebPartyM1" "19" to update de PartyM1 to 19°C
thanks

**Comment by sarnau** [comment]
No, it is strictly read-only. That said: obviously it could be modified to support changing parameters.

**Comment by Pampersfr** [comment]
shame, it's a good progam if you want update it i can testing it ;-)

**Comment by sarnau** [comment]
It was just a proof of concept to show access. I do not need it, so I have to leave it as an exercise for the reader :-)
